pkg/boot/kexec/genpurg.go

```diff
@@ -0,0 +1,95 @@
+//go:build ignore
```

Comment:
Oh I see. Can we put this in one file? Not in 2 files?

Reply:
we can. I just wanted to keep the assembly blah in a separate place, as it is destined for death once I figure out how it has to work.